index.ts

```diff
@@ -108,9 +120,9 @@ export const h = (
 	return element;
 };
 
-// Improve TypeScript support for DocumentFragment
-// https://github.com/Microsoft/TypeScript/issues/20469
+export const Fragment = (typeof DocumentFragment === 'function' ? DocumentFragment : () => {}) as Fragment;
+
 export default {
 	createElement: h,
-	Fragment: typeof DocumentFragment === 'function' ? DocumentFragment : () => {}
+	Fragment
```

Comment:
I'd leave the comment and link here. The underlying issue isn't supported (i.e. `<></>` support)

Reply:
sure, i shouldn't have removed the comment.